### PR TITLE
BPMN2: adaptation for #505

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by vpakan** [comment]
Can one of the admins verify this patch? (use 'ok to test' comment to run verification job)

**Comment by vpakan** [comment]
Can one of the admins verify this patch? (use 'ok to test' comment to run verification job)

**Comment by tsedmik** [comment]
test this

File: plugins/org.jboss.tools.bpmn2.reddeer/src/org/jboss/tools/bpmn2/reddeer/editor/Element.java [hunk 1]
```diff
@@ -539,7 +539,7 @@ private Element addedElement(AbstractEditPart added, String label, ElementType t
 				addedElement.setName(label);
 			}
 			
-		} catch (SWTLayerException e) {
+		} catch (CoreLayerException e) {
```
Comment (apodhrad) [inline plugins/org.jboss.tools.bpmn2.reddeer/src/org/jboss/tools/bpmn2/reddeer/editor/Element.java]: Is this a safe change? Consider using of 
```java
catch (SWTLayerException|CoreLayerException ex)
```

**Comment by apodhrad** [comment]
pushed to jbtis-4.2.x

**Pull request closed without merging**